source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -236,7 +236,21 @@ struct apply_impacts_visitor : public boost::static_visitor<> {
             this->operator()(line);
         }
     }
-    void operator()(const nt::StopArea* ) {
+
+
+    template<typename T>
+    static std::string make_uri(const std::string& ref_uri, T uris) {
```

Comment:
`s/T/const std::map<std::string, T>&/`, it will give better error messages if wrongly used.